sqldelight-gradle-plugin/build.gradle

```diff
@@ -69,9 +69,12 @@ test {
             ":sqldelight-runtime:installLocally",
             ":drivers:android-driver:installLocally",
             ":drivers:sqlite-driver:installLocally",
-            ":sqldelight-runtime:installIosLocally",
-            ":drivers:ios-driver:installLocally",
+//            ":sqldelight-runtime:installIosLocally",
+//            ":drivers:ios-driver:installLocally",
     )
+    useJUnit {
+      excludeCategories 'com.squareup.sqldelight.IosTest'
```

Comment:
I'd prefer leaving this turned on so that when I build on my machine it still runs all tests. Is there something we can do here so that the behavior changes on windows?

Reply:
That part was an accidental check-in, sorry.  I have to manually adjust that in order to build successfully since I don't have an iOS setup.  It'd be nice if there was a way to handle that automatically.